MeetMaryam/blog.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">   
+<link href="C:\Users\yonis\.vscode\School of Code\MeetMaryam\style.css" rel="stylesheet" type="text/css">
+
+<div class="header">
+    <h2>&#10024 Hello I'm Maryam! &#10024</h2>
+</div>
+
+<div class="row">
+    <div class="leftcolumn">
+        <div class="card">
+        <h2>Welcome to my profile</h2>
+        <img src="https://t3.ftcdn.net/jpg/01/76/98/40/360_F_176984023_8I82qQPmKn8TqNAZXIYMCSiwccoUiPBg.jpg" alt="Welcome sign"
+        width="500"
+        height="300">
+        <p>Hello I'm so glad that you have found my blog, I hope you find everything you're looking for.</p>
+    </div>

```

Comment:
We should usually try and format our code consistenly throughout a project. Use a linter such as Prettier to help with this, but aim for best practice without it (and use the linter as a model to work towards first time).

```suggestion
        <div class="card">
            <h2>Welcome to my profile</h2>
            <img src="https://t3.ftcdn.net/jpg/01/76/98/40/360_F_176984023_8I82qQPmKn8TqNAZXIYMCSiwccoUiPBg.jpg" alt="Welcome sign"
        width="500"
        height="300">
            <p>Hello I'm so glad that you have found my blog, I hope you find everything you're looking for.</p>
    </div>

```